docs/website/netcdf-java/ncml/Tutorial.html

```diff
@@ -30,7 +30,7 @@ <h2>Introduction</h2>
   of the functionality in the CDM library. You may also use a specialized XML editor like Oxygen or XML Spy, or an IDE
   like IntelliJ, Eclipse, or NetBeans that have context-sensitive XML editing. </p>
 
-<p>For these exercises, you should <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">run ToolsUI from
+<p>For these exercises, you should <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">run ToolsUI from
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.